fee_allocator/accounting/models.py

```diff
@@ -0,0 +1,40 @@
+from pydantic import BaseModel
+from decimal import Decimal
+from typing import Dict, NewType
+
+
+RawPools = Dict[NewType("PoolId", str), NewType("Label", str)]
+InputFees = Dict[NewType("CorePoolChainName", str), NewType("FeesCollected", int)]
+
+
+class FeeConfig(BaseModel):
+    min_aura_incentive: int
+    min_existing_aura_incentive: int
+    min_vote_incentive_amount: int
+    vebal_share_pct: Decimal
+    dao_share_pct: Decimal
+    vote_incentive_pct: Decimal
+
+
+class RawPoolFeeData(BaseModel):
```

Comment:
Classes like this that contain other classes/complex objects need documentation that explain their purpose and structure.